sapbtp-operator-charts/values.yaml

```diff
@@ -87,3 +87,8 @@ manager:
     enabled: false
 cluster:
   id:
+preDeleteJob:
```

Comment:
please change it to something more generic since we might reuse this image in another job, maybe something like externalImages -> kubectl

Reply:
Done